com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -78,8 +78,16 @@ public partial class HDRenderPipeline
             lightingBuffers.diffuseLightingBuffer = CreateDiffuseLightingBuffer(m_RenderGraph, msaa);
             lightingBuffers.sssBuffer = CreateSSSBuffer(m_RenderGraph, msaa);
 
+            if (m_CurrentDebugDisplaySettings != null && m_CurrentDebugDisplaySettings.data.UseDebugGlobalMipBiasOverride())
```

Comment:
So this isn't enough, you also need to encompass the forward pass.
Then you need to update your test. 
The test should have at least the two type of material we support (i.e forward and deferred), but I will suggest the following list to have a good coverage: 
Lit.shader, shader graph lit, unlit.shader, shadergraph unlit, Axf.shader, ShaderGraph Fabric
+ a Lit Material receiving a Decal.shader and a lit Material receiving a shaderGraph decal.

Lastly you will also need to modifiy the code for cluster decal (and add it to the test, so in addition)
+ a Lit Material **Transparent** receiving a Decal.shader

For the code of cluster decal, please go into DecalUtilities.hlsl and add 
ComputeTextureLOD(...) + _GlobalMipBias


Note that we render normal map during depth prepass to be used by SSR / SSAO. This will be affected as well but I guess it is ok to not test result for those effect in this case.

Reply:
Lastly, but this can done in a different PR, we will need to handle virtual texturing. You can add a bias test in the runtime test either by modifying one of the existing VT test or adding a new one which highlight the blur. 

Thanks

Reply:
> We will also need to update the ShaderGraph texture sampling node to offer an option to not use the bias (i.e generate PLATFORM_SAMPLE_TEXTURE2D instead of SAMPLE_TEXTURE2D )

@sebastienlagarde I have added a new checkbox for texture samplers:
![image](https://user-images.githubusercontent.com/789671/115788099-2f0e8c00-a391-11eb-9b48-5c3954f9f3ad.png)

The code should be on this commit:  https://github.com/Unity-Technologies/Graphics/pull/3829/commits/6108c57bb899afde845313e1a6d4ec6f92253063

@JordanL8 let me know your thoughts on the documentation (leaving a comment might be better for iteration purposes).

Will update the tests soon to also encompass these nodes

Reply:
@cdxntchou has been added to the review for approval.

Reply:
![image](https://user-images.githubusercontent.com/789671/115795540-18bafd00-a39e-11eb-9c35-7ce9a1078394.png)
Hey seb, everything looking good, except that decals do not seam to blend with forward. Wondering if there is something im missing? Or should it work out of the box. LMK if I should dig in a bit more (this happens irrelevant of my changes)

Reply:
Ive highlighted in yello were the decal should be doing something to transparent.